### PR TITLE
Implement spawn protection

### PR DESCRIPTION
**Pull request by ResamVi**
from: https://github.com/yavl/teeworlds-infclassR/pull/173
copy-pasted, not compiled/tested yet

gif doesnt show that the armor count decreases and shows how long one is invincible.
re-entering spawn resets invincible timer

### CODE REVIEW

**Comment by Kaffeine** [comment]
Well, what is about the small games with one zombie and one/two humans?
The alone zombie actually has armor.

**Comment by Kaffeine** [comment]
And hmm, the spawn protection also disables zombies healing (with a hammer) for that period of time.

**Comment by ResamVi** [comment]
I would just remove the feature that spawn protection is temporary and keep it at "inside spawn = no damage". cba with the armor display

File: src/game/server/entities/character.cpp [hunk 1]
```diff
@@ -3685,7 +3685,7 @@ void CCharacter::GrantSpawnProtection()
 	// Indicate time left being protected via eyes
   if(m_ProtectionTick <= 0) {
```
Comment (bretonium) [inline src/game/server/entities/character.cpp:3686]: here two spaces should also be replaced with \t

**Comment by bretonium** [comment]
I've been running it for a while and it looks good so far

**Pull request merged**